api_app/analyzers_manager/file_analyzers/cape_sandbox.py

```diff
@@ -0,0 +1,69 @@
+import time
+
+import requests
+
+from api_app.analyzers_manager.classes import FileAnalyzer
+from tests.mock_utils import MockResponse, if_mock_connections, patch
+
+
+class CAPEsandbox(FileAnalyzer):
+    def set_params(self, params):
+        self.__token = self._secrets["token"]
+        self.__vm_name = params.get("VM_NAME", "win7x64_8")
+
+    def run(self):
+        API_URL: str = "https://www.capesandbox.com/apiv2/tasks/create/file/"
+        headers = {
+            "Authorization": f"Token {self.__token}",
+        }
+
+        files = {
+            "file": (self.filename, self.read_file_bytes()),
+        }
+
+        data = {"machine": self.__vm_name}
+
+        response = requests.post(API_URL, headers=headers, files=files, data=data)
```

Comment:
I think we should add a `response.raise_for_status()` and raise AnalyzerRunException in that case. There are a lot of other analyzers using this if you check